doc/feature/restAPI.md

```diff
@@ -0,0 +1,56 @@
+Publishing the REST API exposed by the HTTP Server
+============
+
+The http server exposes C++ objects via a REST API, in a JSON serialized format.
+The purpose of this document is to describe how this API request is implemented,
+and invoked.
+
+## Requirements
+
+* Define how to invoke the application REST API
+* Implement the JSON representation of the REST API
+
+## API
+
+Invoking a GET resquest on the /api url returns a JSON representation of the
+schemas for all registered serializables.
+
+  {
+    api:
+    {
+        "test/Foo" : "<schema_foo>",
+        "test/Bar" : "<schema_bar>"
+    }
+  }
+
+## Examples
+
+- Create HTTP server and register serializables
+    void setupHTTPServer( const int argc, char** argv )
+    {
+        // Create HTTP server
+        _httpServer = zeq::http::Server::parse( argc, argv );
+        if( !_httpServer )
+            return;
+
+        // Register C++ objects
+        subscribers.push_back( _httpServer );
+        _httpServer.subscribe( camera, LUT, vrParams );
+        _httpServer.register( camera, LUT, vrParams, frame );
+    }
+
+- Query API
+   
+   Request: GET http://hostname:port/api
+   Response: { api: { "object1": "Schema1", "object2": "Schema2" } }
+
+## Implementation
+
+* Add a getJSONSchema public method to Servus::serializable
```

Comment:
I would just call it getSchema, the fact that it's in JSON format is a detail. Please document the C++ API above.